src/L.Control.Heightgraph.js

```diff
@@ -907,13 +932,58 @@ L.Control.Heightgraph = L.Control.extend({
                 this[param].style('display', 'none');
             }
     },
+    /*
+     * Handles the mouseover the map and displays distance and altitude level.
+     * Since this does a lookup of the point on the graph
+     * the closest to the given latlng on the provided event, it could be slow.
+     */
+    _mapMousemoveHandler(evt) {
```

Comment:
This function is also for running in embed mode, when the outer wants to highlight a specific point on the chart on a given event (e.g. mouse over a point on the route).

Reply:
I am just wondering, is this function used? And if yes, where is it used?

Reply:
It's not used by the heightgraph code; it is a way for an external component to tell the heightgraph to highlight a point on the chart.

If your example at https://giscience.github.io/Leaflet.Heightgraph/, if I move the mouse cursor over the route on the map, nothing changes in the heightgraph.

With this function here, the map could handle the mouse over event (e.g. highlight the point on the route) and then call the heightgraph to highlight the corresponding point on the chart.

Let me know if this is still not clear, I could include an example of that.

Reply:
Ah yes I thought so, it would be great if you could include it in the example and maybe add it to the  README as well?

Reply:
Where in the README do you suggest I add that?